### ISSUE TITLE
firefox: JavaScript error: resource://torbutton/modules/tor-control-port.js

### ISSUE THREAD

**Issue by bcHelix** [cypherbits/sandboxed-tor-browser#14]
This might be a FF issue, but googling hasn't really helped. The browser crashes and I get the following when browsing random pages after about 10-15 seconds.

```
firefox: JavaScript error: resource://torbutton/modules/tor-control-port.js, line 232: Error: setevents stream -> 510 Unrecognized command

firefox: JavaScript error: chrome://browser/content/browser.js, line 1078: NS_ERROR_NOT_AVAILABLE: Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIWebNavigation.loadURIWithOptions]

firefox: The futex facility returned an unexpected error code.
```
CSS doesn't seem to be working as well, many of the pages are unformatted with a white background.
I also have a custom user.js file with the following

```
user_pref("browser.tabs.remote.autostart", false);
user_pref("javascript.enabled", false);
user_pref("extensions.torbutton.security_slider", 1);
```
`user_pref("browser.tabs.remote.autostart, false);` is because the tab kept crashing whenever the browser was started, changing it to false solved it. `user_pref("extensions.torbutton.security_slider", 1);` is because the slider would keep resetting from "highest" on every start.

**Comment by cypherbits** [comment]
I can't reproduce your experience, it is working for me.
`firefox: JavaScript error: resource://torbutton/modules/tor-control-port.js, line 232: Error: setevents stream -> 510 Unrecognized command`
This is because inside the sandbox viewing the Tor connection paths is disabled. 
`firefox: JavaScript error: chrome://browser/content/browser.js, line 1078: NS_ERROR_NOT_AVAILABLE: Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIWebNavigation.loadURIWithOptions]`
This error is showing in my case too but the browser works fine for me.
`firefox: The futex facility returned an unexpected error code.`
This is what I think its the cause, but if I don't have more info I can't do anything.

In my case I'm using Ubuntu 18.04 with bubblewrap 0.2.1 and the browser works just fine. The only thing not working is: audio/pulseaudio and seeking in videos (which crashes the tab).

**Comment by bcHelix** [comment]
Yeah I already tried deleting the old one and installing 8.0.1 with the same results. I also tried downgrading bubblewrap to 0.2.1 (which crashes almost instantly) and then to 0.1.7-1 which also crashes.

**Comment by cypherbits** [comment]
Can I have more info about your environment? Like distribution, if its 64bits, etc?

**Comment by bcHelix** [comment]
I'm using a VirtualBox VM running 64-bit Devuan 2.0 Ascii with a custom hardened 4.16 kernel.
